src/components/App/App.js

```diff
@@ -87,6 +90,15 @@ function App() {
             </main>
           </>
         }/>
+        <Route path='/prescription/prescription-not-found' element={
```

Comment:
This doesn't work as a 404 page. Adding the route like this means when the user goes to `/prescription/prescription-not-found` then they will see the 404 page, but it does not show up if the user goes to a URL that does not exist. I pulled the PR down to test it, and the component shows if you go to `/prescription/prescription-not-found` but the 404 page does not show if a route that does not exist is visited.